Dockerfile

```diff
@@ -163,7 +163,7 @@ RUN set -xe \
 # rather than ignoring plattform devs one should use the mediawiki as a base image an copy composer via
 # COPY --from=composer:1 /usr/bin/composer /usr/bin/composer
 # See section PHP version & extensions on https://hub.docker.com/_/composer
-RUN composer install --no-dev --ignore-platform-reqs
+RUN composer install --ignore-platform-reqs
```

Comment:
if removing no-dev affects also the installed extensions (so that dev-requirements are also installed by these), i'd suggest to move the requirements for phpunit to require.